docs/Noise.ipynb

```diff
@@ -0,0 +1,467 @@
+   "source": [
+    "## Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the examples up to now, we've been using `.noise()` to make our queries differentially private. However, there are different types of noise available including Laplacian noise and Gaussian noise. In this notebook, we will describe how the two noise distributions differ and how can one choose which is more appropriate for your particular analysis. \n",
+    "\n",
+    "The default `.noise()` by OpenDP chooses one of these distributions for you based on the limited information you've provided. However, with a better understanding of both distributions, you'll be able to specify the type of noise and its parameters for higher utility!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Laplace vs. Gaussian"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Laplace and Gaussian distributions are the two primary mechanisms to add noise to ensure differential privacy and preserve relationships for statistical analysis. The two distributions may appear similar at a first glance, but there are key differences in their use in differential privacy techniques. \n",
+    "\n",
+    "**1. Definition of Privacy:**\n",
+    " Differential privacy mechanisms using the Laplace distribution satisfy the pure ε-dp, whereas differentially private outputs using the Gaussian mechanism satisfy a more relaxed definition of differential privacy, ε-δ dp. Therefore, Laplace provides stronger privacy guarantees. \n",
+    "\n",
+    "**2. Accuracy:**\n",
+    "Differentially private outputs using the Gaussian mechanism are often further from the true answer compared to outputs using the Laplace mechanism.\n",
+    "\n",
+    "This is supported by the following plot which graphs the emperical probability density function of the Laplace and Gaussian mechanisms with ε = 1 and δ = 10^-5. \n",
+    "\n",
+    "![Histogram of Laplace and Gaussian](https://programming-dp.com/_images/3022199fd49e470e2f86adb6c633df9f586459106067366915cd578cd7e05a5d.png)\n",
+    "\n",
+    "\n",
+    "3. Type of Output: \n",
+    "Gaussian mechanisms perform better when the output is a vector of real numbers rather than a single real number. This is because the Laplace mechanism requires use of the L1 norm (sum of absolute values) and results in a sensitivity of k (length of output vector), whereas the Gaussian mechanism allows the use of the L1 norm or the L2 norm (Euclidean distance). With the Gaussian mechanism and L2 norm, the sensitivty can be $\\sqrt{k}$ In such cases, the L2 sensitivity is significantly lower than the L1 sensitivity. Therefore, the Gaussian mechanism allows for less noise to be added and results in higher utility. \n",
+    "\n",
+    "To learn more about the methodology behind the sensitivity calculation, read [chapter 6](https://programming-dp.com/ch6.html)!\n",
```

Comment:
Agree with Chuck about using newlines, it makes it much easier to review.

The gaussian mechanism doesn't necessarily perform better than the laplace mechanism for a vector-- in the case where the L1 sensitivity is equal to the L2 sensitivity, the laplace mechanism provides overall lower variance.

Gaussian utility is better particularly in the cases where:
1. the L2 sensitivity is smaller than L1 (and discussing when this might be the case would be interesting)
2. there are many compositions